autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py

```diff
@@ -49,8 +47,9 @@ def __init__(self, batch_size: int = 64) -> None:
         self.add_fit_requirements([
             FitRequirement("dataset", (str,), user_defined=True, dataset_property=True),
             FitRequirement("root", (str,), user_defined=True, dataset_property=True),
-            FitRequirement("X_train", (np.ndarray,), user_defined=True, dataset_property=True),
+            FitRequirement("split", (int,), user_defined=True, dataset_property=True),
```

Comment:
Same as above, `split_id` might be more expressive

Reply:
Done!